pkgs/by-name/ni/nixos-rebuild-ng/package.nix

```diff
@@ -69,6 +77,10 @@ python3Packages.buildPythonApplication rec {
 
   pytestFlagsArray = [ "-vv" ];
 
+  makeWrapperArgs = lib.optionals (withTmpdir != null) [
+    "--set TMPDIR ${withTmpdir}"
```

Comment:
Maybe use `--set-default` here? Otherwise users can't set a custom TMPDIR on darwin, while we override platform defaults.

Reply:
Sadly it can't be `--set-default` because `TMPDIR` is set by default in macOS:

```
$ echo $TMPDIR
/var/folders/cn/<random>/T/
```

If you have any other suggestions on how to fix this issue, I am all ears.

Reply:
- macOS users can still override this change with `nixos-rebuild-ng.override { withTmpdir = "/something"; }`
- The only usage of `TMPDIR` is for SSH Control Pane, that should be a socket file (so it should be a small file)